src/main/java/pl/reservo/reservoservice/model/Address.java

```diff
@@ -0,0 +1,29 @@
+package pl.reservo.reservoservice.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+
+
+@Entity
+@Data
```

Comment:
`@Data` not recomended on entities
https://thorben-janssen.com/lombok-hibernate-how-to-avoid-common-pitfalls/

Reply:
Fixed but I am not sure if in correct way